view-transitions/mpa-example-2/page1.css

```diff
@@ -0,0 +1,28 @@
+@view-transition {
+    navigation: auto;
+}
+
+@keyframes slideToRight {
```

Comment:
Could we change this to `slide-to-right` everywhere?
Similarly, `slideFromRight` should be `slide-from-right'.